C#/tests/syntax_test_GeneralStructure.cs

```diff
@@ -1460,12 +1460,26 @@ public class TestModifierOrder
 ///                   ^^^^^^^^^^^^^^^ entity.name.function
 ///                                  ^ punctuation.section.parameters.begin.cs
 ///                                                                               ^ punctuation.section.parameters.end.cs
-    static async Task OnExportCommand(FileInfo outputfile, CancellationToken token) => await Task.CompletedTask;
+    static async Task OnExportCommand(FileInfo outputfile, CancellationToken token) => await Task.CompletedTask {
 /// ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ - invalid
 /// ^^^^^^ storage.modifier
 ///        ^^^^^ storage.modifier
 ///              ^^^^ support.type
 ///                   ^^^^^^^^^^^^^^^ entity.name.function
 ///                                  ^ punctuation.section.parameters.begin.cs
 ///                                                                               ^ punctuation.section.parameters.end.cs
```

Comment:
technically this isn't valid C# code, I would feel happier approving if we changed it to:
``````suggestion
    static async Task OnExportCommand(FileInfo outputfile, CancellationToken token)
/// ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ - invalid
/// ^^^^^^ storage.modifier
///        ^^^^^ storage.modifier
///              ^^^^ support.type
///                   ^^^^^^^^^^^^^^^ entity.name.function
///                                  ^ punctuation.section.parameters.begin.cs
///                                                                               ^ punctuation.section.parameters.end.cs
    {
                await NestedMethod();
```